WebCryptoAPI/derive_bits_keys/derived_bits_length_testcases.js

```diff
@@ -0,0 +1,30 @@
+var testCases = {
+    "HKDF": [
+        {length: 256, expected: algorithms["HKDF"].derivation},
+        {length: 0, expected: undefined}, // not a multiple of 8
```

Comment:
Pedantic nit: 0 is a multiple of 8, but it's disallowed separately.
```suggestion
        {length: 0, expected: undefined}, // should throw an exception
        {length: 1, expected: undefined}, // should throw an exception (not a multiple of 8)
```